src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  gap: 46px 48px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use a consistent order for CSS properties. Consider grouping your properties in the following order: Positioning, Box model, Typography, Visual, Misc. In this case, `display: grid;` should come before `margin:` and `padding:`.